src/beast/python/beast/build/module/Boost.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of Beast: http://github.com/vinniefalco/Beast
+#
+# This file copyright (c) 2015, Tom Ritchford <tom@swirly.com>
+# under the Boost software license http://www.boost.org/LICENSE_1_0.txt
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals)
+
+import os
+from beast.build import Function, Module
+
+def boost(variant, link_libraries):
+    link_libraries = [i if i.startswith('boost_') else 'boost_' + i
+                      for i in link_libraries]
+    try:
+        root = variant.state.environ['BOOST_ROOT']
+    except KeyError:
+        raise KeyError('The environment variable BOOST_ROOT must be set')
```

Comment:
The old version did not raise an error when `BOOST_ROOT` was not present. This is causing problems with travis. We should also update our build docs if we keep this behavior.